### DISCUSSION TITLE
Granular Sharing Permissions for document

### DISCUSSION THREAD

**Discussion by peuportier** [open-webui/open-webui#2244] [Ideas]
Hello, my friend! I hope you are all doing well.

I wanted to suggest an idea that might already be on your development plan: granular sharing permissions for documents.

When a user uploads a document, they should have the option to choose whether to share it or not. Additionally, it would be great to have the ability to reference these documents using hashtags and see them listed separately as shared or non-shared documents when it's calling with "#" 

Thanks you for answers. 

Wish you a good day

**Comment by simonaszilinskas** [comment]
great suggestion, had the same ideas !

**Comment by EurekaChen** [comment]
so, will this suggestion be consider?
